src/main/java/com/minecolonies/coremod/util/LanguageHandler.java

```diff
@@ -31,19 +34,50 @@ private LanguageHandler()
      * @param key    unlocalized key.
      * @param args   Objects for String.format().
      */
-    public static void sendPlayerLocalizedMessage(@NotNull final EntityPlayer player, final String key, final Object... args)
+    public static void sendPlayerLocalizedMessage(@NotNull final EntityPlayer player, final String key, final String... args)
     {
-        sendPlayerMessage(player, format(key, args));
+        sendPlayerMessage(player, key, args);
     }
 
     /**
      * Send a message to the player.
      * @param player the player to send to.
+     * @param key the key of the message.
      * @param message the message to send.
      */
-    public static void sendPlayerMessage(@NotNull final EntityPlayer player, final String message)
+    public static void sendPlayerMessage(@NotNull final EntityPlayer player, final String key, final Object... message)
     {
-        player.addChatComponentMessage(new TextComponentString(message));
+        TextComponentTranslation translation = null;
+        for(Object object: message)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'object' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)